### PR TITLE
[TASK] Document the new classes

### LINKED ISSUE #666: Document CssInliner in the README

### PR DESCRIPTION
**Pull request by oliverklee**
Fixes #666

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -259,17 +258,39 @@ will be converted to `width="100"`.
...
 ```
 
 ### Technology preview of new classes
 
 Currently, a refactoring effort is underway, aiming towards replacing the
 grown-over-time `Emogrifier` class with the new `CssInliner` class and moving
-additional HTML processing into separate `CssProcessor` classes (which will
-inherit from `AbstractHtmlProcessor`). You can try the new classes, but be
-aware that the APIs of the new classes still are subject to change. 
+additional HTML processing into separate `CssProcessor` classes (which
```
Comment (JakeQZ) [inline README.md]: Though this was present previously, I'm not sure about presenting a non-existent class name `CssProcessor` formatted as code, and in any case it should be HTML (not CSS) Processor.  Is the message conveyed just as well if the invented class name is simply dropped, and perhaps the bit in brackets moved out of brackets, i.e.

> ... and moving additional HTML processing into separate classes which inherit from `AbstractHtmlProcessor`.

?

File: README.md [hunk 2]
```diff
@@ -259,17 +258,39 @@ will be converted to `width="100"`.
...
+inherits from `AbstractHtmlProcessor`). You can try the new classes, but be
```
Comment (JakeQZ) [inline README.md]: should be "inherit" (3rd person plural - a class inherits, but classes inherit).

File: README.md [hunk 3]
```diff
@@ -259,17 +258,39 @@ will be converted to `width="100"`.
...
+```
+
+You can also have the `CssToAttributeConverter` work on a `DOMDocument`:
+
+```php
+$converter = \Pelago\Emogrifier\HtmlProcessor\CssToAttributeConverter::fromDomDocument($domDocument);
 $visualHtml = $converter->convertCssToVisualAttributes()->render();
 ```
 
 ### Technology preview of new classes
```
Comment (JakeQZ) [inline README.md]: Again, this was in 2.1, but I wonder if it should be clearer that this applies to all the classes in this section, "Processing HTML".  I guess we'll find out after 2.2...

File: README.md (hunk 2 above)
Comment (oliverklee) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (oliverklee) [reply README.md]: Sounds good. Done.

File: README.md (hunk 3 above)
Comment (oliverklee) [reply README.md]: I've rearranged the sections a bit now.

**Comment by oliverklee** [comment]
Changed and repushed.

**Pull request merged**